docs/src/Microphysics1M.md

```diff
@@ -263,6 +263,41 @@ In other derivations cloud ice, similar to cloud liquid water,
 
      - Do we want to test different size distributions?
 
+Here we plot the Marshall-Palmer particle size distribution for 4 different values for the rain specific humidity (q_rai).
+We also show the aspect ratio of snow particles.
+
+```@example
+include("plots/MarshallPalmer_distribution.jl")
+```
+![](MarshallPalmer_distribution.svg)
+
+## Rain radar reflectivity
```

Comment:
I would vote to put this chapter at the end of the docs page, just before the example figures. Because it's a property that we diagnose based on the parameterization, and not a prognostic variable